parse_lib.py

```diff
@@ -95,8 +94,9 @@ def resolve_relative_resource_urls(html_string):
     imgs = html.find_all("img", src=True)
     equations = html.find_all("object", data=True)
     list(map(update_anchor_href, anchors))
+    equations_as_imgs = list(map(partial(convert_to_img, html), equations))
+    imgs.extend(equations_as_imgs)
     list(map(partial(resolve_resource, 'src'), imgs))
```

Comment:
Ditto for this line.  In fact, assuming we reach consensus on this, perhaps a quick find/replace is in order for the whole parser code base, since it is public now.